src/ulohy/brainfuck.md

```diff
@@ -0,0 +1,49 @@
+# Překladač jazyka Brainfuck
+
+Jazyk [Brainfuck](https://esolangs.org/wiki/Brainfuck) je velmi jednoduchý esoterický programovací
+jazyk obsahující pouze osm operátorů, lineární paměť a adresu aktuální paměťové buňky. Jedna buňka
+odpovídá jednomu bytu.
+
+* Operátor `+` inkrementuje hodnotu aktuální buňky.
+* Operátor `-` dekrementuje hodnotu aktuální buňky.
+* Operátor `>` inkrementuje adresu buňky (po inkrementaci ukazujeme na následující buňku).
+* Operátor `<` dekrementuje adresu buňky (po dekrementaci ukazujeme na předcházející buňku).
+* Operátor `[` uvozuje začátek cyklu. Cyklus probíhá, dokud hodnota buňky adresované v době
+  vyhodnocování podmínky není nulová (adresovanou buňku můžeme měnit uvnitř cyklu).
+* Operátor `]` představuje konec těla cyklu, ale nereprezentuje žádné chování.
+* Operátor `,` přečte jeden byte ze vstupu a uloží hodnotu do aktuálně adresované buňky.
+* Operátor `.` vypíše hodnotu aktuálně adresované buňky jako ASCII znak na standardní výstup.
+
+Všechny ostatní znaky jsou ignorovány.
+
+Naprogramujte interpret jazyka Brainfuck. Interpret (angl. *interpreter*), je překladač, který
```

Comment:
I would rather say that it executes the code, rather than compiles/translates it.